lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,58 @@ module ActiveModel
   class Serializer
     class Adapter
       class Attributes < Adapter
+        attr_accessor :_nested_associations
```

Comment:
If we need to refactor the attributes adapter, I'm good with that in this PR or another